Ladybird/Qt/WebContentView.cpp

```diff
@@ -599,14 +602,17 @@ void WebContentView::update_palette(PaletteMode mode)
     client().async_update_system_theme(make_system_theme_from_qt_palette(*this, mode));
 }
 
-void WebContentView::create_client()
+void WebContentView::initialize_client(WebView::ViewImplementation::CreateNewClient create_new_client)
 {
-    m_client_state = {};
+    if (create_new_client == CreateNewClient::Yes) {
```

Comment:
Why do we only handle this in the Qt chrome?

Reply:
because that's what I tested with and it took forever already to test this and get the Tab refactoring correct-ish. :sweat_smile: . I was hoping to do the other chromes in a follow-up, or yakbait someone else into doing the updates.

Reply:
Gotcha, just slap a FIXME in the other chromes for now then?

Reply:
The FIXMEs ended up in the `Ladybird: Add IPC call for creating a new child tab ` commit like so: 

```
  m_web_view_bridge->on_new_web_view = [self](auto activate_tab, auto, auto) {
        // FIXME: Create a child tab that re-uses the ConnectionFromClient of the parent tab
        return [self.observer onCreateNewTab:"about:blank"sv activateTab:activate_tab];
    };
```
But it probably makes sense to add some into the initialize_client commit as well.